services/api-db/docker-entrypoint-initdb.d/01-migrations.sql

```diff
@@ -1737,6 +1737,16 @@ CREATE OR REPLACE PROCEDURE
   END;
 $$
 
+CREATE OR REPLACE PROCEDURE
+  clean_stale_project_data()
+
+  BEGIN
+    DELETE FROM env_vars WHERE project NOT IN (SELECT id FROM project);
```

Comment:
do we also need to delete env_vars where the environment no longer exists (but the project still does?)

Reply:
Yes :doh: